ci/tests/vmachine/vmachine_test.py

```diff
@@ -104,16 +121,38 @@ def create_machine_from_existing_raw_disk(machine_name, vpool_name, disk_name):
         logger.error("Error while creating vmachine: {0}".format(err))
 
 
-def remove_machine_by_name(vmachine_name):
-    logger.info("Removing {0} vmachine".format(vmachine_name))
-    out, err = general.execute_command('virsh destroy {0}'.format(vmachine_name))
+def shut_down_vmachine_by_name(machine_name):
```

Comment:
Imo this is also not a good approach
We should try to work with objects as much as possible
So the machine object should be passed
The function name is not a good choice either, it should just be shutdown_vmachine and start_vmachine